core/src/main/java/org/elasticsearch/action/support/replication/ReplicationRequest.java

```diff
@@ -177,6 +177,11 @@ public void writeTo(StreamOutput out) throws IOException {
         out.writeString(index);
     }
 
+    @Override
+    public Task createTask(long id, String type, String action) {
+        return new ReplicationTask(id, type, action, this::getDescription, getParentTaskNode(), getParentTaskId());
```

Comment:
`this::getDescription` is a method reference rather than a direct call. This change just changes the class built. We've been storing the reference to getDescription for a week or so now....